scripts/bench/evict.py

```
@@ -0,0 +1,12 @@
+import os, sys
+def evict(p):
+    try:
+        fd = os.open(p, os.O_RDONLY)
+        try: os.posix_fadvise(fd, 0, 0, os.POSIX_FADV_DONTNEED)
+        finally: os.close(fd)
+    except OSError as e: print("evict fail", p, e, file=sys.stderr)
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)